package.json

```diff
@@ -62,6 +62,7 @@
     "@msgpack/msgpack": "^1.12.2",
     "@openzeppelin/contracts": "^3.2.0",
     "@servie/events": "^1.0.0",
+    "@solana/web3.js": "^0.87.0",
```

Comment:
hi @developerfred I'm sorry but you need to rebase this PR (with pnpm) and add this dependency to `packages/maskbook/package.json`